src/docfx/build/xref/XrefMapLoader.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Docs.Build
+{
+    internal class XrefMapLoader
+    {
+        private static byte[] s_uidBytes = Encoding.UTF8.GetBytes("uid");
+
+        public static ListBuilder<(string, Lazy<IXrefSpec>)> Load(string filePath)
```

Comment:
consider return `List<>` over `ListBuilder<>`

Reply:
You are right, no need to use `ListBuilder` here, no concurrency involved